forge.config.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 const fs = require('fs');
 const packageJson = require('./package.json');
+const root = process.cwd();
```

Comment:
Very minor nit, but can we move this down to be in the section on lines 6-7 instead? Usually the section of requires at the top of the file only contains requires.

Reply:
Cool!